tsl/test/expected/continuous_aggs_errors.out

```diff
@@ -404,7 +404,7 @@ ALTER VIEW :"PART_VIEW_SCHEMA".:"PART_VIEW_NAME" SET(timescaledb.refresh_lag = '
 ERROR:  cannot alter the internal view of a continuous aggregate
 \set ON_ERROR_STOP 1
 DROP TABLE conditions CASCADE;
-NOTICE:  drop cascades to 4 other objects
+NOTICE:  drop cascades to 6 other objects
```

Comment:
What additional objects do we have now given that nothing else changed here?

Reply:
the cagg view itself depends on additional objects now if its a union view

Reply:
@svenklemm I am missing something here. We always had the mat hypertable , raw hypertable and internal view. This PR only adds a UNION ?